src/drt/src/dr/FlexDR_init.cpp

```diff
@@ -3485,29 +3463,14 @@ void FlexDRWorker::initMarkers(const frDesign* design)
 
 void FlexDRWorker::init(const frDesign* design)
 {
-  // if initDR
-  //   get all instterm/term for each net
-  // else
-  //   1. get all insterm/term based on begin/end of pathseg, via
-  //   2. union and find
-  //
-  // using namespace std::chrono;
-  initMarkers(design);
-  if (getDRIter() && getInitNumMarkers() == 0 && !needRecheck_) {
-    skipRouting_ = true;
-  }
-  if (skipRouting_) {
-    return;
-  }
-  initFixedObjs(design);
```

Comment:
Where did this functionality go?

Reply:
initFixedObjs was removed since fixed_objs_ are not needed dr. initMarkers is done in main() and distributedMain()

Reply:
Just remove it

Reply:
what do you mean? I already removed it.